### ISSUE TITLE
Dashboard: undefined is not an object (evaluating 'D.find(t=>t.value===e).label')

### ISSUE THREAD

**Issue by alexcican** [umami-software/umami#2632]
### Describe the Bug

When I go to the [dashboard](https://analytics.eu.umami.is/dashboard) I get this error:

```
Something went wrong.
undefined is not an object (evaluating 'D.find(t=>t.value===e).label')
renderValue@https://analytics.eu.umami.is/_next/static/chunks/7672-e3aad413a6ecb579.js:1:5451
@https://analytics.eu.umami.is/_next/static/chunks/6dfc6103-f4ad57155da9942d.js:1:125336
rk@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:40399
lF@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:57311
o4@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:94662
@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:94484
o3@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:94489
oQ@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:91978
n2@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:37412
n4@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:38147
@https://analytics.eu.umami.is/_next/static/chunks/fd9d1056-29288d683efce69d.js:1:39180
``` 

<img width="1027" alt="Screenshot 2024-03-26 at 17 25 22" src="https://github.com/umami-software/umami/assets/1185882/9ea989bb-a5ff-43f7-b750-cdb2dd8212d0">

System: macOS Sonoma 14.4 (23E214)
Browser: Safari Technology Preview Release 190 (Safari 17.4, WebKit 19619.1.5.5.2)

Can you fix it please?

### Database

Umami Cloud

### Relevant log output

```shell
[Error] TypeError: undefined is not an object (evaluating 'D.find(t=>t.value===e).label')
  (anonymous function) (8069-d806ffd01109fbb6.js:1:3798)
  lE (fd9d1056-29288d683efce69d.js:1:56292)
  (anonymous function) (fd9d1056-29288d683efce69d.js:1:56752)
  ng (fd9d1056-29288d683efce69d.js:1:28515)
  ny (fd9d1056-29288d683efce69d.js:1:28608)
  a$ (fd9d1056-29288d683efce69d.js:1:73053)
  aV (fd9d1056-29288d683efce69d.js:1:73699)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73411)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73974)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73974)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73411)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73411)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73743)
  (anonymous function) (fd9d1056-29288d683efce69d.js:1:107291)
  o7 (fd9d1056-29288d683efce69d.js:1:107837)
  n2 (fd9d1056-29288d683efce69d.js:1:37541)
  n4 (fd9d1056-29288d683efce69d.js:1:38148)
  (anonymous function) (fd9d1056-29288d683efce69d.js:1:39181)
[Error]  (2)
TypeError: undefined is not an object (evaluating 'D.find(t=>t.value===e).label')
"
c@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:30271
s@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:30429
O@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:24060
p@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:18795
m@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:21497
P@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:22535
a@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:36710
R@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:24214
o@https://analytics.eu.umami.is/_next/static/chunks/7546-730a85fc49e3ae73.js:1:2734
f@https://analytics.eu.umami.is/_next/static/chunks/app/layout-5772d9a66f55ed1c.js:1:771
ng@https://analytics.eu.umami.is/_next/static/chunks/6dfc6103-f4ad57155da9942d.js:1:62423
u@https://analytics.eu.umami.is/_next/static/chunks/4997-2b6eba51fb6caf78.js:24:24752
IntlProvider@https://analytics.eu.umami.is/_next/static/chunks/7546-730a85fc49e3ae73.js:1:13112
v@https://analytics.eu.umami.is/_next/static/chunks/app/layout-5772d9a66f55ed1c.js:1:1265
x@https://analytics.eu.umami.is/_next/static/chunks/app/layout-5772d9a66f55ed1c.js:1:1512
body
html
c@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:33682
s@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:33770
A@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:10915
s@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:18082
p@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:18795
I@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:15232
w@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:4452
x
T@https://analytics.eu.umami.is/_next/static/chunks/8069-d806ffd01109fbb6.js:1:4801"
  (anonymous function) (8069-d806ffd01109fbb6.js:1:3798)
  a (layout-5772d9a66f55ed1c.js:1:681)
  componentDidCatch (7546-730a85fc49e3ae73.js:1:1848)
  (anonymous function) (fd9d1056-29288d683efce69d.js:1:56860)
  ng (fd9d1056-29288d683efce69d.js:1:28515)
  ny (fd9d1056-29288d683efce69d.js:1:28608)
  a$ (fd9d1056-29288d683efce69d.js:1:73053)
  aV (fd9d1056-29288d683efce69d.js:1:73699)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73411)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73974)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73974)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73411)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73411)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73374)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:74047)
  a2 (fd9d1056-29288d683efce69d.js:1:84276)
  aV (fd9d1056-29288d683efce69d.js:1:73743)
  (anonymous function) (fd9d1056-29288d683efce69d.js:1:107291)
  o7 (fd9d1056-29288d683efce69d.js:1:107837)
  n2 (fd9d1056-29288d683efce69d.js:1:37541)
  n4 (fd9d1056-29288d683efce69d.js:1:38148)
  (anonymous function) (fd9d1056-29288d683efce69d.js:1:39181)
```

### Which browser are you using? (if relevant)

Safari Technology Preview Release 190 (Safari 17.4, WebKit 19619.1.5.5.2)

**Comment by mikecao** [comment]
Try clearing your browser local storage and see if that helps.

**Comment by franciscao633** [comment]
Fixed in release v2.11.0
